.github/workflows/ci.yml

```diff
@@ -277,6 +277,41 @@ jobs:
         working-directory: ./flutter/realm_flutter/tests
 
 # Windows jobs
+  baas-windows:
+    runs-on: ubuntu-latest
```

Comment:
this is seems strange. is this intentional

Reply:
What is strange - the fact we're adding baas tests for Windows or that the deploy step runs on ubuntu?

Reply:
I misread this. Closing